modules/logging/logging.go

```diff
@@ -54,3 +54,14 @@ func (u *LogStruct) MethodInfo(model string, method string, stack ...bool) *logr
 		"method": method,
 	})
 }
+
+func (u *LogStruct) PanicRecover(requestID string) *logrus.Entry {
```

Comment:
__[golint]__
* [golint] exported method LogStruct.PanicRecover should have comment or be unexported